opus/application/static_media/js/hash.js

```diff
@@ -155,10 +155,14 @@ var o_hash = {
                         case "order":
                             opus.prefs[slug] = value.split(',');
                             break;
-                        case "startobs" || "cart_startobs":
+                        case "startobs":
```

Comment:
you can make this a fall-through case so you don't have to duplicate code, that's why I suggested the code:
case "startobs":
case "cart_startobs":
    do stuff();
    break;

Doing it that way makes it clear to all that read that those two cases are handled the same.  So it's not that your code is wrong, but if they are always the same, it will be clearer in the long run to make it obvious in the code.

Reply:
New commit has been updated to have fall-through case as suggested, thanks for the feedback.